packages/transformers/elm/src/ElmTransformer.js

```diff
@@ -0,0 +1,125 @@
+// @flow
+
+import {Transformer} from '@parcel/plugin';
+import {sync as commandExists} from 'command-exists';
+import path from 'path';
+import fs from 'fs';
+import spawn from 'cross-spawn';
+import elm from 'node-elm-compiler';
+import {inject as injectElmHMR} from 'elm-hot';
+import {minify} from 'terser';
+import ThrowableDiagnostic from '@parcel/diagnostic';
+
+export default new Transformer({
+  async getConfig({asset, options}) {
+    const installPackage = async name => {
+      if (!options.autoinstall) {
+        throw new ThrowableDiagnostic({
+          diagnostic: {
+            message: `Denpendency '${name}' is not installed and autoinstall is turned off. Either install dependency manually or enable autoinstall`,
```

Comment:
If you do `await options.packageManager.resolve('elm', asset.filePath, {autoinstall: options.autoinstall});` this will handled automatically,